code/modules/client/preferences/middleware/keybindings.dm

```diff
@@ -81,6 +79,7 @@
 
 /datum/asset/json/keybindings
 	name = "keybindings"
+	early = TRUE
```

Comment:
Why bother? It doesn't need to be.

Reply:
It's required by the prefs menu, see my comment below

Reply:
It's needed by prefs menu but it doesn't actually need to be in early assets because it doesn't do anything other than read a GLOB